README.md

```diff
@@ -1 +1,272 @@
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Html structure with images, icons, header & menu-bar, links, project, skills, etc]**
+- **[Styling css with colors, paddings, margins & sizes, etc]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+_ Opend CMD, Terminal or GitBash in your pc and type this command _ git clone git@github.com:myaccount/my-project.git_
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+
+Install this project with:
+_ You can simply download this project you don't need to install in your pc because it is webpage.
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+_ You can use it by download in your computer.
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML
+- CSS
+
+## <img src="https://scontent.fisb6-1.fna.fbcdn.net/v/t39.30808-1/244664695_100276912439806_3650167602209431250_n.jpg?stp=dst-jpg_p200x200&_nc_cat=111&ccb=1-7&_nc_sid=7206a8&_nc_eui2=AeHOoS20TXUIBI-cKLdq7MyDYfDGj-ys0nFh8MaP7KzScV84n31f_TIamvcf7BRqujzs8jBIhO4AIjnmOxsw6aFp&_nc_ohc=S8v8SjRuzX8AX9Ic0VN&_nc_ht=scontent.fisb6-1.fna&oh=00_AfAN8Wp-6faZ5xU0_Wa00X4SwPTDWgCh1N3eq1-Uska-eg&oe=63C2EE23" border-radisu="50%" width="30px" style="border-radius: 50%;"> Authors <a name="authors"></a>
+👤 **Syed Atril**
+
+- GitHub: [@githubhandle](https://github.com/Atril33)
+- Twitter: [@twitterhandle](https://twitter.com/AtrilSyed)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/syed-atril-831696248/)
```

Comment:
[OPTIONAL] Kindly correct the handles again. The should look like this in the code:
```
- GitHub: [@Atril33](https://github.com/Atril33)
- Twitter: [@Atril33](https://twitter.com/AtrilSyed)
```
Then it will result like this:
```
GitHub: @Atril33
Twitter: @Atril33
LinkedIn: Atril33
```